Migrate tests and docs to cli v3.1 (#415)
kyma-project/docker-registry: docs/user/tutorials/01-20-expose-registry.md

## Prerequsities

* [kubectl](https://kubernetes.io/docs/tasks/tools/)

> * [kyma CLI](https://github.com/kyma-project/cli)

Reviewer: ```suggestion
* [Kyma CLI](https://github.com/kyma-project/cli)
```

    EOF
    ```

>    Once the DockerRegistry CR becomes `Ready`, you see a Secret name which you need to use as `ImagePullSecret` when scheduling workloads in the cluster.

Reviewer: ```suggestion
   Once the DockerRegistry CR becomes `Ready`, you will see a Secret name that you must use as `ImagePullSecret` when scheduling workloads in the cluster.
```
